scene/level_one.py

```diff
@@ -47,10 +58,30 @@ def all_fg_elements(self):
 
     def add_fg_element(self, value):
         self.fg_elements.append(value)
+        if hasattr(value, 'collider'):
+            self.colliders.append(value.collider)
 
     def remove_fg_element(self, element):
         if element in self.fg_elements:
             self.fg_elements.remove(element)
+        if hasattr(element, 'collider') and element.collider in self.colliders:
+            self.colliders.remove(element.collider)
+
+    def add_obstacles(self):
+        w, h = TURKEY_IMAGE.get_size()
+        # NOTE: place obstacles in 5x5 grid
+        x_step = state.SCREEN_WIDTH // 5
+        y_step = state.SCREEN_HEIGHT // 5
+        for grid_x in range(0, state.SCREEN_WIDTH, x_step):
+            for grid_y in range(0, state.SCREEN_HEIGHT, y_step):
+                # NOTE: skip the square the player is spawned in
+                if grid_x == 2 * x_step and grid_y == 2 * y_step:
+                    continue
+                x = grid_x + randint(0, x_step - 1)
+                y = grid_y + randint(0, y_step - 1)
+                angle = randint(0, 359)
+                rot_v = randint(-45, 45)
```

Comment:
This allows for turkey legs that aren't spinning, if that is important

Reply:
yeah that's intentional, the why of turkey leg spins is a mystery to all